File: jats-demo-webapp/src/main/java/com/github/pepe79/jats/demo/model/Product.java
package com.github.pepe79.jats.demo.model;

import java.util.List;

public class Product
{
	private int id;

	private List<ProductVariant> variants;

	public List<ProductVariant> getVariants()
	{
		return variants;
	}

	public void setVariants(List<ProductVariant> variants)
	{
		this.variants = variants;
	}

	public int getId()
	{
		return id;
	}

	public void setId(int id)
	{
		this.id = id;
	}

}

File: jats-js/src/test/ts/demo/model/Product.ts
class Product
{
	private variants : ProductVariant[];

	private id : number;

	public getVariants() { return this.variants; }
	public setVariants( variants : ProductVariant[] ) { this.variants = variants; }

	public getId() { return this.id; }
	public setId( id : number ) { this.id = id; }


}
